Tasks/UsePythonV1/task.json

```diff
@@ -0,0 +1,73 @@
+{
+    "id": "33C63B11-352B-45A2-BA1B-54CB568A29CA",
+    "name": "UsePython",
+    "friendlyName": "Use Python",
+    "description": "Sets up Python environment.",
+    "helpUrl": "https://go.microsoft.com/fwlink/?linkid=871498",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=871498)",
+    "category": "Tool",
+	"ecosystem": "python",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 151,
+        "Patch": 0
+    },
+    "demands": [],
+    "instanceNameFormat": "Use Python $(versionSpec)",
+    "groups": [
+        {
+            "name": "advanced",
+            "displayName": "Advanced",
+            "isExpanded": false
+        }
+    ],
+    "inputs": [
+        {
+            "name": "version",
+            "type": "string",
+            "label": "Version",
+            "required": false,
+            "defaultValue": "3.x",
+            "helpMarkDown": "Version range or exact version of a Python version to use, using semver's version range syntax. [More information](https://go.microsoft.com/fwlink/?LinkID=2006180)"
+        },
+        {
+            "name": "architecture",
+            "type": "pickList",
+            "label": "Architecture",
+            "defaultValue": "x64",
+            "required": true,
+            "helpMarkDown": "The target architecture (x86, x64) of the Python interpreter.",
+            "groupName": "advanced",
+            "options": {
+                "x86": "x86",
+                "x64": "x64"
+            }
+        }
+    ],
+    "outputVariables": [
+        {
+            "name": "pythonLocation",
+            "description": "The directory of the installed Python distribution. Use this in subsequent tasks to access this installation of Python."
+        }
+    ],
+    "execution": {
+        "Node": {
```

Comment:
Fyi, make sure you update this to `Node10` as you're testing and update the task-lib to v3.0.0-preview. I ran into some issues with tests failing on one handler and not the other, we want to make sure we're testing in the right environment and we want to make sure we're using the Node 10 handler.